test/api.rb

```diff
@@ -10,27 +10,27 @@ def test_email(*)
   end
 end
 
-class ApiJob < ActiveJob::Base
+class ApiAjJob < ActiveJob::Base
```

Comment:
I renamed this job to allow for `ApiWorker` to be named `ApiJob` - since this is an active job I used `Aj` to differentiate the two.

Other suggestions are appreciated.